apps/test-tsx-app/admin.test.ts

```diff
@@ -0,0 +1,50 @@
+import test, { describe } from 'node:test';
+import { getPriciSdk, updateAdminUi } from './utils/prici-runner.js';
+import assert from 'node:assert';
+
+const authToken =
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiSm9obiBEb2UiLCJpYXQiOjE1MTYyMzkwMjJ9.e4vybEZSMruiGm8XgTxlD2QDGuOBz5l3gRfmjlmb1PY';
+
+describe('admin api route', async () => {
+  test('should throw 401 exception when ADMIN_UI is invalid', async () => {
```

Comment:
We cannot block the admin on authentication header, because it's not something the browser can send.

Mayr we need another auth mechanism to support the admin.

Reply:
This is what I currently get when I do these requests. I leave it as is unless you tell me otherwise or we handle it in a later PR as you said.